src/System.Diagnostics.Process/tests/ProcessStandardConsoleTests.cs

```diff
@@ -43,22 +43,12 @@ public void TestChangesInConsoleEncoding()
 
             try
```

Comment:
The current test already testing our change without need to register the encoding provider. what exactly the case you are asking to test?

---
In reply to: [93728474](https://github.com/dotnet/corefx/pull/14649#discussion_r93728474) [](ancestors = 93728474)